lib/oneview-sdk/resource/api200/login_detail.rb

```diff
@@ -0,0 +1,26 @@
+# (C) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative 'resource'
+
+module OneviewSDK
+  module API200
+    # Login Details resource implementation
+    class LoginDetail < Resource
```

Comment:
Since this is inheriting from Resource, it should likely def the methods it will not be using as `unavailable_method`, examples include: `create` & `delete`.

Reply:
It might be better to just drop the inheritance here though, since this resource will likely not be able to use any normal methods.